bridges/SamsungMobileChangelogBridge.php

```diff
@@ -0,0 +1,99 @@
+<?PHP
+
+class SamsungMobileChangelogBridge extends BridgeAbstract
+{
+    const NAME = 'Samsung Mobile Changelog';
+    const URI = 'https://doc.samsungmobile.com/';
+    const DESCRIPTION = 'Returns the changelog of selected devices from the Samsung Mobile documentation in English';
+    const MAINTAINER = 'ajain-93';
+    const PARAMETERS = [
+        [
+            'devices' => [
```

Comment:
```suggestion
            'device' => [
```
This is about one model, so I would rename this field to `device`. Please adapt all uses as well.